nettrine/app/Bootstrap.php

```diff
@@ -24,6 +24,12 @@ public static function boot(): Configurator
 		$configurator
 			->addConfig(__DIR__ . '/config/common.neon');
 
+		$localNeon = __DIR__ . '/config/local.neon';
+		if (file_exists($localNeon)) {
```

Comment:
local.neon bude vzdycky potreba, netreba to podminkovat na file_exists

Reply:
ok, podmínku smažu